students/visokoo/lesson03/assignment/src/basic_operations.py

```diff
@@ -0,0 +1,161 @@
+"""
+    A compilation of methods for interacting with an ORM DATABASE
+"""
+
+import logging
+from playhouse.shortcuts import model_to_dict
```

Comment:
What is this module for?

Reply:
Brought in that module to convert a model object from peewee to a dict.